plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/HCLAppScanIASTReader.java

```diff
@@ -114,33 +143,12 @@ private void parseFindings(TestSuiteResults tr, String json) throws Exception {
     }
 
     private int cweLookup(String rule) {
-        switch (rule) {
```

Comment:
Some of the cases are not present in your new map. For backwards compatibility, it should at least contain all old entries.

Reply:
Added the relevant cases to the map

Reply:
@darkspirit510 - You OK with this now so I should merge?

Reply:
@davewichers Can you merge it please?